OSDOCS-18179-networking-bug-batch-4-22 release notes (#112557)
openshift/openshift-docs: modules/rn-ocp-release-notes-known-issues.adoc

* Currently, the `topo-aware-scheduler` provided by the NUMA Resources Operator (NRO) does not support Kubernetes priority-based preemption. When all NUMA zones on available nodes are fully consumed by lower-priority pods, a high-priority pod with a `PreemptLowerPriority` policy remains in `Pending` state indefinitely instead of preempting the lower-priority pods. As a consequence, workloads that depend on priority-based preemption for scheduling recovery do not function correctly when using the `topo-aware-scheduler`. (link:https://issues.redhat.com/browse/OCPBUGS-77930[OCPBUGS-77930])

* {product-title} does not support restoring volume snapshots in a topology domain that does not have access to the datastore where the snapshot resides. You must manually schedule pods that use a persistent volume claim (PVC) that restore a snapshot to a region and zone with the snapshot. Using a shared datastore across all regions and zones meets this requirement. (link:https://issues.redhat.com/browse/OCPBUGS-84702[OCPBUGS-84702])

> * In some {azure-first} configurations ({azure-short} private, public {azure-short} {product-title}, and private {azure-short} {product-title}), outbound connectivity is achieved through outbound rules that are connected to the backend address pool of the primary IP address of the VM network interface controller (NIC). Before this update, the Egress IP address was added to the public load balancer backend address pool when an `OutBoundRule` parameter was not specified. The Egress IP addresses are no longer added to the public load balancer backend pool for any {product-title} cluster hosted on {azure-short}, regardless of the existence of an `OutBoundRule` parameter. As a result, Egress IP addresses will have no outbound connectivity except for the infrastructure subnet in an {azure-short} {product-title} cluster. (link:https://redhat.atlassian.net/browse/OCPBUGS-57447[OCPBUGS-57447])

Reviewer: Move 57447 to the bug fix section?

Author: It says in the Jira that it is a "Known Issue"

Author: Do you think it belongs in bug fix section. Thanks

Reviewer 2: https://docs.redhat.com/en/documentation/openshift_container_platform/4.21/html/release_notes/ocp-4-21-release-notes#zstream-4-21-16_release-notes:~:text=In%20some%20Microsoft,OCPBUGS%2D77154)

This belongs in KIs.

Reviewer 2: I will have 2 possibly 3 more to add to this by EOD, fyi.

> * When switching between PTP profiles, metrics from the previously applied profile might not be cleaned up correctly. This issue is most noticeable when changing network interfaces (NICs). As a workaround, you can restart the pod to clean up the metrics. (link:https://issues.redhat.com/browse/OCPBUGS-66413[OCPBUGS-66413])

Reviewer: ```suggestion
* When switching between PTP profiles, metrics from the previously applied profile might not be cleaned up correctly. This issue is most noticeable when changing network interfaces (NICs). To work around this problem, you can restart the pod to clean up the metrics. (link:https://issues.redhat.com/browse/OCPBUGS-66413[OCPBUGS-66413])
```

<img width="945" height="273" alt="Image" src="https://github.com/user-attachments/assets/66795299-a3be-4cbc-80b3-62c997a02ee1" />

Author: Done
